Documentation/articles/getting_started/1_creating_a_new_project_vs.md

```diff
@@ -1,19 +1,75 @@
 # Creating a Project with Visual Studio
 
-Start up Visual Studio and select **New Project...** in the upper left corner.
+MonoGame is available on Windows for development using the instructions below to get started with MonoGame 3.8.
+
+> Note, MonoGame 3.8 projects are no longer compatible with earlier versions of MonoGame.  If you wish to work on or build older MonoGame projects, then you will still need to install [MonoGame 3.7.1](https://www.monogame.net/downloads/) or earlier to open them.
```

Comment:
This isn't true, MG < 3.8 is compatible with 3.8+. It's just the setup process that's changed.

Reply:
Not true.  Without the Monogame 3.7.1 MSI install, this is what I get when I open 3.7 solutions with only the Monogame 3.8 templates installed
![image](https://user-images.githubusercontent.com/1793042/89084882-f3f2a580-d388-11ea-9726-5f53d748ba1d.png)

Reply:
Ah, I misunderstood what you meant by incompatible. Yes, old projects require the SDK install. But I wouldn't call old projects incompatible with 3.8, because upgrading is easy (and I'd say recommended to get all .NET Core goodies).

But... this part of the guide is for setting up a new project, which I would very, very strongly recommend everyone does with 3.8 in the future. I don't think we need to bring up the older versions or how things changed here.

I would move this information to the migration guide. Just a short note, something like:

> If you do not want to upgrade your project you'll need to keep the MonoGame SDK installed.

When we announce the release we should be clear that people need to check out the migration page if they have old projects and want to upgrade to 3.8.

Reply:
Ok, that's fair enough. But we need to ensure the message is clear.  (and that include our own samples until I get them updated, soon (tm) )